src/App.tsx

```diff
@@ -1,18 +1,51 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+// eslint-disable-next-line object-curly-newline
+import { Route, Routes, useMatch, useNavigate } from 'react-router-dom';
```

Comment:
Don't use eslint-disable unless necessary.

```suggestion
import {
  Route,
  Routes,
  useMatch,
  useNavigate
} from 'react-router-dom';
```